src/content/quickstart/verify_with_discovery.md

```diff
@@ -5,16 +5,9 @@ To verify the deployment follow the steps below.
 export KUBECONFIG=cluster-b/auth/kubeconfig
 kubectl -n default create deployment nginx --image=nginxinc/nginx-unprivileged:stable-alpine
 kubectl -n default expose deployment nginx --port=8080
-kubectl -n default apply -f - <<EOF
```

Comment:
This line should be deleted as well

Reply:
This is deleted, it not present on the right side.